strings/tips.txt

```diff
@@ -0,0 +1,189 @@
+As the IAA, try to negotiate with the Warden if sentences seem too high for the crime.
+As the IAA, you can try to convince the captain and Head of Security to hold trials for prisoners in the courtroom.
+As the Head of Personnel, you are not higher ranking than other heads of staff, even though you are expected to take the Captain's place first should he go missing. If the situation seems too rough for you, consider allowing another head to become temporary Captain.
+As the Head of Personnel, you are just as large a target as the Captain because of the potential power your ID and computer can hand out.
+As the Mime, your invisible wall power blocks people as well as projectiles. You can use it in a pinch to delay your pursuer.
+As the Mime, your oath of silence is your source of power. Breaking it robs you of your powers and of your honor.
+As the Clown, if you lose your banana peel, you can still slip people with your PDA! Honk!
+As the Clown, your Grail is the mineral bananium, which can be given to the Roboticist to build you a fun and robust mech beloved by everyone.
+As the Clown, you can use your stamp on a sheet of cardboard as the first step of making a honkbot. Fun for the whole crew!
+As the Clown, spice your gimmicks up! Nobody likes a one-trick pony.
+As the Chaplain, your null rod has anti magic functions: it can destroy cultist runes by hitting them, and it makes you immune to cult magic.
+As the Chaplain, your bible is also a container that can store small items.
+As the Chaplain, you are much more likely to get a response by praying to the gods than most people. To boost your chances, make altars with colorful crayon runes, lit candles, and wire art.
+As a Botanist, you can hack the MegaSeed Vendor to get access to more exotic seeds. These seeds can alternatively be ordered from cargo.
+As a Botanist, you can mutate the plants growing in your hydroponics trays with unstable mutagen or, as an alternative, crude radioactives from chemistry to get special variations.
+As a Botanist, you should look into increasing the potency of your plants. This increases the size, amount of chemicals, points gained from grinding them in the biogenerator, and lets people know you are a proficient botanist.
+As a Botanist, you can combine production trait chemicals just like a Chemist. Chlorine (blumpkin) + radium and phosphorus (glowshrooms) equals unstable mutagen!
+As a Cook, you can create a very wide variety of food with the crafting menu. You can find it by looking for the hammer icon near your intents.
+As a Cook, you can rename your custom made food with a pen.
+As the Bartender, you can use a circular saw on your shotgun to make it easier to store.
+As a Janitor, if someone steals your janicart, you can instead use your space cleaner spray, grenades, water sprayer, exact revenge or order another from Cargo.
```

Comment:
Remove `exact revenge` not only because it encourages tiding, but also because it is horrendous grammar.

Reply:
fixed in the next commit